CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 - Add `default_cooldown` and `health_check_grace_period` options to workers ASG (by @ArieLevs)
 - Fix support for ASG max instance lifetime for workers (by @barryib)
 - Add support for envelope encryption of Secrets (by @babilen5)
+- Added support for EKS 1.15 (by @sc250024)
```

Comment:
Please mark it as `breaking`

Reply:
Yep, no problem!

Reply:
Done here => https://github.com/terraform-aws-modules/terraform-aws-eks/pull/776/commits/a1c5510c2193b5f53302a45b4c759a9c5dc53bfb